.github/workflows/vulnerability-scan.yml

```diff
@@ -0,0 +1,58 @@
+name: "Security vulnerability scan"
+
+on:
+  schedule:
+    - cron: "20 02 * * *"
+  workflow_dispatch:
+
+permissions:
+  contents: read
+
+env:
+  GO_VERSION: '1.20'
+
+jobs:
+  go:
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-go@v4
+        with:
+          go-version: ${{ env.GO_VERSION }}
+          check-latest: true
+      - name: Scan
+        run: make scan-go
+
+  node:
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-go@v4
+        with:
+          go-version: ${{ env.GO_VERSION }}
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 18
+      - name: Scan
+        run: make scan-node
+
+  java:
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-go@v4
+        with:
+          go-version: ${{ env.GO_VERSION }}
+      - uses: actions/setup-java@v3
+        with:
+          java-version: 17
+          distribution: temurin
+          cache: maven
+      - name: Scan
+        run: make scan-java
+      - name: "Archive dependency-check report"
```

Comment:
Should Go and node also upload a scan report?

Reply:
They don’t generate any report file, only outputting to the console, which can be viewed in the build log. Only the Java scan actually outputs a separate report with more information than output to the console.

Reply:
Aha, that makes sense then! :)